src/views/Collectibles/components/NftList.tsx

```diff
@@ -6,21 +6,19 @@ import { fetchWalletNfts } from 'state/collectibles'
 import { useGetCollectibles } from 'state/collectibles/hooks'
 import NftCard from './NftCard'
 import NftGrid from './NftGrid'
-import EasterNftCard from './NftCard/EasterNftCard'
-import LotteryNftCard from './NftCard/LotteryNftCard'
 
 /**
  * A map of bunnyIds to special campaigns (NFT distribution)
  * Each NftCard is responsible for checking it's own claim status
  *
  */
 const nftComponents = {
-  'easter-storm': EasterNftCard,
-  'easter-flipper': EasterNftCard,
-  'easter-caker': EasterNftCard,
-  lottie: LotteryNftCard,
-  lucky: LotteryNftCard,
-  baller: LotteryNftCard,
+  'easter-storm': NftCard,
+  'easter-flipper': NftCard,
+  'easter-caker': NftCard,
+  lottie: NftCard,
+  lucky: NftCard,
+  baller: NftCard,
```

Comment:
Just remove these. It will use the default card in the render component.

Reply:
Keep the empty object though

Reply:
Done